src/features/tags/hooks/useTag.ts

```diff
@@ -0,0 +1,53 @@
+import { useSelector, useStore } from 'react-redux';
+
+import { loadItemIfNecessary } from 'core/caching/cacheUtils';
+import { RootState } from 'core/store';
+import { ZetkinTag } from 'utils/types/zetkin';
+import { tagAssigned, tagLoad, tagLoaded, tagUnassigned } from '../store';
+import { useApiClient, useEnv } from 'core/hooks';
+
+interface UseTagReturn {
+  assignToPerson: (personId: number, value?: string) => void;
+  removeFromPerson: (personId: number) => Promise<void>;
+  tagData: ZetkinTag | null;
+  tagError: unknown | null;
+  tagIsLoading: boolean;
```

Comment:
The name of the hook already tells us that the hook will return a tag, so there is no need for these prefixes. I would call these properties:

```suggestion
  data: ZetkinTag | null;
  error: unknown | null;
  isLoading: boolean;
```